src/NaNMath.jl

```diff
@@ -61,6 +61,45 @@ function sum(x::AbstractArray{T}) where T<:AbstractFloat
     return result
 end
 
+"""
+NaNMath.median(A)
+
+##### Args:
+* `A`: An array of floating point numbers
+
+##### Returns:
+*   Returns the median of all elements in the array, ignoring NaN's.
+    Returns NaN for an empty array or array containing NaNs only.
+
+##### Examples:
+```julia
```

Comment:
This package doesn't have doctests, but could you format this example in that style? E.g., https://juliadocs.github.io/Documenter.jl/stable/man/doctests/#REPL-Examples